### ISSUE TITLE
prefixParameter time scale is wrong in all but the first parameter (_1)

### ISSUE THREAD

**Issue by matteobachetti** [nanograv/PINT#883]
If one adds the following line at glitch.py:180

```
            print(gnepnm, glep.quantity._time.__dict__)
```
it comes out that the timescale is "tdb" only in the first of the parameters in the series (e.g. "GLEP_1", not "GLEP_2")
I discovered this by working on a piecewise timing model

**Comment by luojing1211** [comment]
How do you get the glep in the print?
-Jing

**Comment by matteobachetti** [comment]
I ran `pytest -svv tests/test_glitch.py`, and adding the line above produces lots of output of the kind
```
INFO: 0.4130984484746796   [pint.models.glitch]
GLEP_1 {'_scale': 'tdb', 'precision': 3, '_in_subfmt': '*', '_out_subfmt': '*', '_jd1': array(2454953.), '_jd2': array(0.42239), 'cache': defaultdict(<class 'dict'>, {'format': {'pulsar_mjd_long': 54952.92239}})}
INFO: 0.0   [pint.models.glitch]
GLEP_2 {'_scale': 'utc', 'precision': 3, '_in_subfmt': '*', '_out_subfmt': '*', '_jd1': array(2455463.5), '_jd2': array(0.89923953), 'cache': defaultdict(<class 'dict'>, {'format': {'pulsar_mjd_long': 55463.89923952912795}})}
INFO: 0.3989054835547666   [pint.models.glitch]
GLEP_3 {'_scale': 'utc', 'precision': 3, '_in_subfmt': '*', '_out_subfmt': '*', '_jd1': array(2456369.5), '_jd2': array(0.56142733), 'cache': defaultdict(<class 'dict'>, {'format': {'pulsar_mjd_long': 56369.56142733058368}})}
```

**Comment by paulray** [comment]
Wow, that looks wrong!

**Comment by luojing1211** [comment]
That will be in the prefixParameter.new_par() method. Somehow the new
prefix parameter did not get the right time scale.
-Jing

**Comment by matteobachetti** [comment]
Resolved in #884
